modules/tooltip/react/lib/Tooltip.tsx

```diff
@@ -5,8 +5,19 @@ import {TransformOrigin, getTranslateFromOrigin} from '@workday/canvas-kit-react
 import {keyframes} from '@emotion/core';
 
 export interface TooltipProps {
+  /**
+   *  Origin from which the Tooltip will animate from.
+   * @default {horizontal: 'center', vertical: 'top'}
+   */
   transformOrigin: TransformOrigin;
+  /**
+   * Unique id of the Tooltip.
+   */
   id?: string;
+  /**
+   * The content for the Tooltip.
+   */
+  children?: string;
```

Comment:
Should we include this prop?

Reply:
What was the intent behind adding it? We have quite a few places like this where we are omitting inherited props that are critical to functionality (`children` is very common, `value` for inputs, etc.). We should probably have a consistent stance.

Reply:
Yeah I wasn't 100% if this should be included. Hence the comment. 

My perspective was we document it for `Toast` which is `The message text for the Toast.`. 

Now thinking about it more I think we should omit `children` from the docs unless there are restrictions on the `children` type.